applications/luci-app-tor-hs/htdocs/luci-static/resources/view/tor_hs_setting.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+'require view';
+'require ui';
+'require form';
+
+return view.extend({
+	formdata: { torhs: {} },
+
+	render: function(data) {
+		var m, s, o;
+
+		m  = new form.Map('tor-hs', _('Hidden service configuration'));
+		m.readonly=false;
+
+		o = m.section(form.TableSection, "hidden-service");
+		o.option(form.Value, "Name", _("Name"));
+		o.option(form.Value, "Description", _("Description"));
+		o.option(form.Flag, "Enabled", _("Enabled"));
+		o.option(form.Value, "IPv4", _("IPv4"));
+		o.datatype = 'ip4addr("nomask")';
+		o.option(form.DynamicList, "PublicLocalPort", _("Public Local Port"));
+		o.option(form.Value, "HookScript", _("HookScript"));
+		o.anonymous = true;
+		o.addremove = true;
```

Comment:
Translation should use a `'` and not `"`
String should be also use `'` and not `"` ( for example: `"HookScript"` -> `'HookScript'` )